### DISCUSSION TITLE
可以用VMware在centos7上启动dperf并测试连接win上的java应用程序吗

### DISCUSSION THREAD

**Discussion by jiangtao960** [baidu/dperf#439] [Q&A]
这是我的配置文件
![1713167079986](https://github.com/baidu/dperf/assets/167066405/377160d4-bd37-42e2-996b-c21535c60c04)
这是运行dperf后的报文
![image](https://github.com/baidu/dperf/assets/167066405/0d0a9f1f-282d-4576-8968-b390988c2889)
VMware和java应用在同一台机器上，ping和curl都没问题，但用dperf连接java应用收不到请求，望指点

**Comment by pengjianzhang** [comment]
44.1 44.2分别是什么

**Reply by jiangtao960** [reply]
44.1是本机VMnet8 IPv4地址
![image](https://github.com/baidu/dperf/assets/167066405/62cf3bc0-5527-4ec4-ab99-f5577881e970)
44.2是虚拟机设置的默认网关地址
![image](https://github.com/baidu/dperf/assets/167066405/63a0af93-5ad0-422d-ae5a-51e15fa1ed5e)

**Comment by pengjianzhang** [comment]
配置修改一下
port xxxx 192.168.44.3  192.168.44.1 

不确定dperf是否支持vmware的网卡

**Reply by jiangtao960** [reply]
配置如下时dperf启动报错
![image](https://github.com/baidu/dperf/assets/167066405/20be2b6c-c6e5-4c73-bc83-e4b290444120)
![image](https://github.com/baidu/dperf/assets/167066405/904b0cf8-0cb6-430b-9cc3-0990a1d8e47d)

如果是因为网卡问题，有可以参考的文档推荐下吗

**Reply by pengjianzhang** [reply]
你读一下配置文档 配置问题

**Reply by pengjianzhang** [reply]
https://dperf.org/doc/html/configuration#section9
